oidcclient/jwk_test.go

```diff
@@ -132,8 +137,9 @@ func TestOIDCTenant_ReadJWKs(t *testing.T) {
 			name: "read from token keys endpoint with deleted zone",
```

Comment:
Test name still talks about zones. Could you please also replace all remaining "zone" occurrences in the project? (except from the deprecated func of course)

Reply:
removed all mentions of zone